Source/DataSources/BillboardVisualizer.js

```diff
@@ -162,6 +164,35 @@ define([
     };
 
     /**
+     * Computes a bounding sphere which encloses the visualization produced for the specified entity.
+     *
+     * @param {Entity} entity The entity whose bounding sphere to compute.
+     * @param {BoundingSphere} result The bounding sphere onto which to store the result.
+     * @returns {BoundingSphereState} BoundingSphereState.DONE if the result contains the bounding sphere,
+     *                       BoundingSphereState.PENDING if the result is still being computed, or
+     *                       BoundingSphereState.FAILED if the entity has no visualization in the current scene.
+     */
+    BillboardVisualizer.prototype.getBoundingSphere = function(entity, result) {
```

Comment:
Given the fish we have to fry, I would not jump on the `Advanced` namespace.

Given the cost associated with making things public (doc, tests, forum support, deprecation, etc.), I would only make things public that we know will be useful.  If we are planning a overhaul here, I would make this `@private` so it's not something that we introduce knowing that it will go away.  From what we are trying to achieve now with the entity API, it feels like an implementation detail.